src/Components/SystemDetail/RecommendationsTable.scss

```diff
@@ -2,15 +2,35 @@
   tr {
     --pf-c-table--cell--FontSize: var(--pf-global--FontSize--md);
   }
-  .newline {
-    white-space: pre-line;
-  }
-  .tab {
-    white-space: pre-wrap;
-  }
   .recs-table-empty {
     .pf-c-empty-state__icon {
       fill: var(--pf-global--success-color--100);
     }
   }
 }
+.newline {
+  white-space: pre-line;
+}
+.tab {
+  white-space: pre-wrap;
+}
+Th {
+  font-weight: normal;
+}
+strong {
+  padding: 12px;
+}
+.table-border-top {
+  border-top: 1px solid #d9d9d9;
+}
+.detected-issues-table {
+  margin-left: 3%;
+  width: 40%
+}
+.suggestions-table {
+  margin-left: 3%;
+  width: 90%;
+}
+.margin-text-bottom {
+  margin-bottom: 2%;
+}
```

Comment:
@r14chandra I tried to add the media query at the end and it worked for me. Here is the code for same:

`@media (max-width: 768px ){
  .detected-issues-table {
    width: 90%
  }
}`

Try it and let me know if it is working for you as well!